parity-crypto/src/publickey/ec_math_utils.rs

```diff
@@ -148,4 +149,24 @@ mod tests {
 
 		assert_eq!(negation, public);
 	}
+
+	#[test]
+	fn known_public_is_valid() {
+		let public = Random.generate().unwrap().public().clone();
+		assert!(public_is_valid(&public));
+	}
+
+	#[test]
+	fn generation_point_expected() {
+		let point = generation_point();
+		assert_eq!(format!("{:x}", point), "79be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798483ada7726a3c4655da4fbfc0e1108a8fd17b448a68554199c47d08ffb10d4b8");
+	}
+
+	#[test]
+	fn public_multiplication_verification() {
+		let secret = Secret::from_str("a100df7a048e50ed308ea696dc600215098141cb391e9527329df289f9383f65").unwrap();
+		let mut public = generation_point();
+		public_mul_secret(&mut public, &secret).unwrap();
+		assert_eq!(format!("{:x}", public), "8ce0db0b0359ffc5866ba61903cc2518c3675ef2cf380a7e54bde7ea20e6fa1ab45b7617346cd11b7610001ee6ae5b0155c41cad9527cbcdff44ec67848943a4");
```

Comment:
If this value is from a spec somewhere it'd be great to document that. :)

Reply:
It's just a randomly created secret